### ISSUE TITLE
explorer.app:280 attempt to call global 'systemInfo' (a nil value)

### ISSUE THREAD

**Issue by JasonTheKitten** [nothjarnan/axiom#46]
Files>Menu>Programs
explorer.app:280 attempt to call global 'systemInfo' (a nil value)
![image](https://user-images.githubusercontent.com/15697938/71767260-5f9a9000-2ed8-11ea-9951-ce770c751d54.png)

**Comment by JasonTheKitten** [comment]
While at it, there is this other error too
![image](https://user-images.githubusercontent.com/15697938/71767275-8658c680-2ed8-11ea-98d8-7051282c71f5.png)

**Comment by nothjarnan** [comment]
I'll have a look.

**Comment by LDDestroier** [comment]
What's weird is that `terminal()` and `sysInfo()` are never defined in any of Axiom's files. My guess is that they were put there as placeholders, and were then overlooked.

**Comment by nothjarnan** [comment]
You're correct to an extent. They were moved into individual files (see Axiom/programs). And `sysInfo()` was removed shortly after version 1.0, when it was still sitting in my Dropbox. I probably have a legacy implementation of it laying around that could be used to replace it.
